src/Form/Type/ApiMediaType.php

```diff
@@ -68,18 +67,6 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $provider->buildMediaType($builder);
     }
 
-    /**
-     * {@inheritdoc}
-     *
-     * NEXT_MAJOR: remove this method.
-     *
-     * @deprecated Remove it when bumping requirements to Symfony >=2.7
-     */
-    public function setDefaultOptions(OptionsResolverInterface $resolver)
```

Comment:
Removing a public method breaks BC.

Reply:
I'll take a look later in the other PR about public functions